pkg/apiserver/handlers.go

```diff
@@ -152,6 +152,7 @@ func RecoverPanics(handler http.Handler) http.Handler {
 				http.StatusNotFound,
 				http.StatusUnauthorized,
 				http.StatusForbidden,
+				http.StatusNotModified,
```

Comment:
This protects against 304's and is probably fine.

I was hoping to not log on *any* response that's just being echoed by our proxy, though. Which is harder.:(